common/src/main/java/org/conscrypt/ConscryptEngine.java

```diff
@@ -210,6 +212,13 @@ private static NativeSsl newSsl(SSLParametersImpl sslParameters, ConscryptEngine
         }
     }
 
+    /**
+     * Configures the default {@link BufferAllocator} to be used by all instances.
```

Comment:
Let's say "all future instances", since it won't affect previously-created instances.